unit-tests-jest/numbers.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers) {
+  if (!Array.isArray(numbers)) {
```

Comment:
Nice edge case check, but you should `throw new Error(message);` instead of returning a string.

You can test that using the `toThrow` matcher.